lib/v2/youtube/maintainer.js

```diff
@@ -6,4 +6,5 @@ module.exports = {
     '/playlist/:id/:embed?': ['HenryQW'],
     '/subscriptions/:embed?': ['TonyRL'],
     '/user/:username/:embed?': ['DIYgod'],
+    '/live/:username/:embed?': ['sussurr127'],
```

Comment:
```suggestion
    '/live/:username/:embed?': ['sussurr127'],
    '/playlist/:id/:embed?': ['HenryQW'],
    '/subscriptions/:embed?': ['TonyRL'],
    '/user/:username/:embed?': ['DIYgod'],
```
Insert new entries in alphabetical order.